classes/checkout-gateway.php

```diff
@@ -14,7 +14,7 @@
 use WC_Order;
 use WC_Payment_Gateway;
 use Exception;
-use Settings;
+use Zaver\Classes\Settings;
```

Comment:
I believe the namespace the rest of the classes uses in that folder are just under Zaver, so lets keep it the same for now until we move them all into the src folder and change the naming of the files to match the PSR-4 standard